chore: bump version to 0.3.3

saravanan7mani7 opened the pull request:
- Updates bs58check from ^2.1.2 to ^3.0.0 to resolve crypto dependency issues
- Updates uglify-js from ^3.4.7 to ^3.19.0 for ES6+ compatibility
- Bumped to 0.3.3

feri42 commented on dist/bchaddrjs-0.3.2.js:
A better way to handle this kind of change is to create a separate commit that renames `bchaddrjs-0.3.2.js` to ``bchaddrjs-0.3.3.js` and then copy over the new content. That way we can easily review the changes.

feri42 reviewed (commented):
We use a loose conventional commit spec for PR titles and commit messages: https://coda.io/d/Engineering-Handbook_dWMI1pUBpME/Welcome-to-Exodus_sulD_Mly?docIds%5B0%5D=WMI1pUBpME&search=conventional%20commit#Conventional-commits_tuZOTNwH

Updated the PR title.

saravanan7mani7 committed:
chore: update bs58check to ^3.0.0 and uglify-js to ^3.19.0

- Updates bs58check from ^2.1.2 to ^3.0.0 to resolve crypto dependency issues
- Updates uglify-js from ^3.4.7 to ^3.19.0 for ES6+ compatibility

ChALkeR commented on README.md:
```diff
@@ -29,7 +29,7 @@ $ bower install --save bchaddrjs
 ### Manually
 
-You may also download the distribution file manually and place it within your third-party scripts directory: [dist/bchaddrjs-0.3.2.min.js](https://cdn.rawgit.com/bitcoincashjs/bchaddrjs/master/dist/bchaddrjs-0.3.2.min.js).
+You may also download the distribution file manually and place it within your third-party scripts directory: [dist/bchaddrjs-0.3.3.min.js](https://cdn.rawgit.com/bitcoincashjs/bchaddrjs/master/dist/bchaddrjs-0.3.3.min.js).
```
this is not relevant
we are in the fork, the link doesn't make sense, undo readme changes

ChALkeR commented on README.md:
```diff
@@ -49,7 +49,7 @@ You may include a script tag in your HTML and the `bchaddr` module will be defin
   <head>
     ...
-    <script src="https://cdn.rawgit.com/bitcoincashjs/bchaddrjs/master/dist/bchaddrjs-0.3.2.min.js"></script>
+    <script src="https://cdn.rawgit.com/bitcoincashjs/bchaddrjs/master/dist/bchaddrjs-0.3.3.min.js"></script>
```
ditto

ChALkeR commented on dist/bchaddrjs-0.3.2.js:
dist changes are also not relevant, feel free to just delete that dir in a separate commit
or ignore dist completely

saravanan7mani7 committed:
chore: bump version to 0.3.3

feri42 commented:
@ChALkeR good now?
